src/App.tsx

```diff
@@ -1,61 +1,142 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { FormEventHandler, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user: User;
+};
+
+type TodoFromServer = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+};
+
+let maxId = 0;
+
+const createTodoArray = (todo: TodoFromServer): Todo => {
```

Comment:
this function is not creting array, rename this.Its only changes your incoming todo